src/oemof/solph/components/_converter.py

```diff
@@ -117,8 +113,12 @@ def __init__(
             label=label,
             inputs=inputs,
             outputs=outputs,
-            **custom_attributes,
+            custom_properties=custom_attributes,
```

Comment:
@p-snft why aren't these parameter-names aligned? I guess it started [here](https://github.com/oemof/oemof-network/pull/40) Wouldn't it be best to change in `oemof.network` to `custom_attributes` as well, especially as `custom-attributes` are frequently used in the `oemof-tabular` API?!

Reply:
In Python, an attribute is a fixed term referring to a variable defined by the class in the scope of an object. The implementation that some though we had (but never had) in network added attributes

```python
my_node = Node("name", some_feature= 5)  # API was never implemented
my_node.some_feature # "some_feature" is an attribute, API was never implemented
# 5
```

However, the API now is
```python
my_node = Node("name", custom_properties={"some_feature": 5})
my_node.custom_properties["some_feature"]  # "some feature" is not an attribute
# 5
```

Reply:
well `custom_properties` is an attribute though :laughing: